models/User.js

```diff
@@ -60,4 +60,9 @@ userSchema.methods.getPic = function() {
   return pic;
 };
 
+userSchema.methods.getRecasedEmailAddress = function() {

```

Comment:
This should help to fix a small bug causing Teams links to not function as seamlessly as possible.